src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -478,7 +407,7 @@ public DataAccessRequest cancelDataAccessRequest(String referenceId) {
         List<DataSet> dataSetsToApprove = dataSetDAO.
                 findNeedsApprovalDataSetByDataSetId(datasetIdsForDatasetsToApprove);
 
-        // Sort documents by sort time, create time, then reversed.
+        // Sort dars by sort time, create time, then reversed.
         Comparator<DataAccessRequest> sortField = Comparator.comparing(d -> d.getData().getSortDate());
         Comparator<DataAccessRequest> createField = Comparator.comparing(d -> d.getData().getCreateDate());
```

Comment:
Not for this PR, but we should be sorting by the dates on the dar, not the dar data.

Reply:
And it should be update date, then create date, then reversed.